setup.cfg

```diff
@@ -2,4 +2,16 @@
 universal=1
 
 [flake8]
+exclude=*migrations*
 ignore=E122,E123,E124,E125,E126,E127,E128,E129,E131,E201,E202,E203,E221,E225,E226,E231,E241,E251,E261,E262,E265,E271,E301,E302,E303,E501,E502,E701,E713,E714,F401,F403
+max-line-length = 99
```

Comment:
Here's the Django team's conclusion from our docs, "Don’t limit lines of code to 79 characters if it means the code looks significantly uglier or is harder to read. We allow up to 119 characters as this is the width of GitHub code review; anything longer requires horizontal scrolling which makes review more difficult. This check is included when you run flake8. Documentation, comments, and docstrings should be wrapped at 79 characters, even though PEP 8 suggests 72."

Of course, this project can take a different stance, but I think you'll end up doing a lot of needless refactoring to get things down to 79 (you could easily run a quick test to determine how many additional lines would need to be modified).